File: Python/Gate-Two/LEVEL 4/TaskZero.py
def addition(num1, num2):
    return num1 + num2



def elements(numbers):

    total = 0

    for num in numbers:

        total += num

    return total

def largest(numbers):

    result = 0

    numbers.sort

    result = numbers[-1]

    return result



def largest_sort(numbers):

    largest = 0

    for num in numbers:

        if num > largest:

            largest = num

    return largest




def smallest_sort(numbers):

    smallest = numbers[0]

    for num in numbers:

        if num < smallest:

            smallest = num

    return smallest





def even_numbers(numbers):

    for num in numbers:

        if num % 2 == 0:

            return num




def reverse_numbers(numbers):

    reverse_list = []


    for num in range(len(numbers) -1, -1, -1):

        reverse_list.append(numbers[num])

    return reverse_list

#    return numbers[::-1]



def length_numbers(numbers):

    count = 0;

    for num in numbers:
        count += 1
    return count

#or return len(numbers)


def average_numbers(numbers):

    average = 0;

    total = 0

    for num in numbers:
      
        total += num

        average = total / 5

    return average

#or return len(numbers)

File: Java/Gate-Two/LEVEL 4/LEVEL 4 TEST/TaskZero.java
public class TaskZero{


    public static int addition(int num1, int num2){

        return num1 + num2;



}



    public static int totalSum(int [] numbers){

        int total = 0;

        for(int count = 0; count < numbers.length; count++){

            total += numbers[count];
            

}

            return total;

}



    public static int largestSorting(int [] numbers){

        int largest = 0;

        for(int count = 0; count < numbers.length; count++){
    
            if(numbers[count] > largest){
                
               largest = numbers[count];

}

}

            return largest;

}



    public static int smallestSorting(int [] numbers){

        int smallest = numbers[0];

        for(int count = 0; count < numbers.length; count++){
    
            if(numbers[count] < smallest){
                
               smallest = numbers[count];

}

}

            return smallest;

}





    public static int evenNumbers(int [] numbers){

        int even = 0;

        for(int count = 0; count < numbers.length; count++){
    
            if(numbers[count] % 2 == 0){
                
               even = numbers[count];

}

}

            return even;

}





    public static int [] reverseNumbers(int [] numbers){


        int [] reverse = new int[numbers.length];

        int index = 0;

        for(int count = numbers.length - 1; count >= 0; count--){
    
  
               reverse[index] = numbers[count];

                index ++;
}


        return reverse;

}



    public static int lengthNumbers(int [] numbers){


         int total = 0;

        for(int count = 0; count <= numbers.length; count++){
    
            total = count;
}


        return total;

}




}
